.github/workflows/alert-test.yml

```diff
@@ -14,19 +14,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
       - name: Install Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v3
         with:
-          version: v3.4.0
-
-      - name: Install yq
```

Comment:
See https://github.com/chrisdickinson/setup-yq/pull/9 and the action has not been updated since March 2021